### ISSUE TITLE
Self hosting cloud image/deployment setup - prebuilt images or setups

### ISSUE THREAD

**Issue by daluu** [typpo/textbelt#119]
Now that the public service is no longer free, it would be nice when one can deploy a self hosted version easily w/ minimal setup (e.g. set/modify config, then boot up and deploy, no need do all the prerequisite installations).

And this would be especially nice when users can deploy this to cloud services so that they don't have to run locally/internally (not everyone has their own servers or knows how to deploy locally on demand when they want to send some text messages).

This can be building upon issue #88. But expanding to cloud services like say:

* a prebuilt Amazon AWS EC2 AMI image to deploy on EC2
* docker image for issue 88 prebuilt and pushed to public Docker repository to deploy on any cloud (or internally) where docker can run
* a prebuilt OpenShift container configuration to deploy
* some Google Cloud config?
* you get the idea...

Basically there are some cloud services with (limited) free tier that users can deploy the self hosted service to. e.g. AWS free tier (1 year only), Google Cloud free tier, OpenShift free tier.

**Comment by zdods** [comment]
@daluu I'm all for this, getting this to even run locally has been a nightmare. (even with having all the dependencies installed and working)

**Comment by daluu** [comment]
@zdods, what did you run/deploy it on? Could use your setup as a reference for preconfigured deployment.

**Comment by zdods** [comment]
@daluu its been ages ha! I think I ended up just messing about local and not going much farther than that.

These days honestly a Docker image with the correct config variables exposed would be enough for me - most projects I work on are in k8s clusters so an image like that would be perfect.

**Comment by ryanckulp** [comment]
@zdods hey Zach, i just made a simple (+ free) self deploy guide here: https://github.com/typpo/textbelt/issues/176
